virttest/utils_split_daemons.py

```diff
@@ -419,3 +419,25 @@ def is_modular_daemon(session=None):
         else:
             IS_MODULAR_DAEMON[host_key] = False
     return IS_MODULAR_DAEMON[host_key]
+
+
+def adopt_func_on_libvirtd(func):
+    """
+        Decorator method to handle process in different daemon modes
+
+        :params func: function to be wrapped
+    """
+
+    def signal_libvirtd(*args, **kwargs):
+        """
+        Wrap method of the decorator
+
+        :params *args: arguments of the function
+        :params **kwargs: dict args of the function
+        """
+        if is_modular_daemon() and args[0] == 'libvirtd':
+            list_args = list(args)
+            list_args[0] = 'virtqemud'
+
+        return func(*list_args, **kwargs)
```

Comment:
it looks like there is potential risk here, if is_modular_daemon() return false,  call func(*list_args, **kwargs) may throw list_args is unknown variable error. Therefore, pulling list_args = list(args) out of scope of if statemnet could be one way to fix this risk

Reply:
Updated.